src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.Runtime.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime</PackageId>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Runtime API.</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.LUIS</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;LUIS;Language Understanding Intelligent Service;Language Understanding;REST HTTP client</PackageTags>
     <PackageReleaseNotes>
```

Comment:
Please update the `PackageReleaseNotes` here

Reply:
@dsgouda  what should be in the PackageReleaseNotes ?

Reply:
What changes you are planning to release in this version. Please check the 
"Changes in this release:" section